base/accumulate.jl

```diff
@@ -269,10 +269,10 @@ julia> accumulate(+, fill(1, 2, 5), dims=2, init=100.0)
  101.0  102.0  103.0  104.0  105.0
 ```
 """
-function accumulate(op, A; dims::Union{Nothing,Integer}=nothing, kw...)
+function accumulate(op, A; dims::Union{Nothing,Integer}=nothing, kws...)
     if dims === nothing && !(A isa AbstractVector)
         # This branch takes care of the cases not handled by `_accumulate!`.
-        return collect(Iterators.accumulate(op, A; kw...))
+        return collect(Iterators.accumulate(op, A; kws...))
     end
     nt = values(kw)
```

Comment:
This will error now. A change like this feels a bit scary since we don't have 100% test coverage. (Personally I think changes like this are rather pointless unless 1) this will be included in a style guide for JuliaLang/julia and 2) will be enforced in code review.)

Reply:
1) Yes, I'm looking through `grep -Ir '[^a-z_:0-9]kw[^a-z_0-9]' base stdlib test` now, and then I'll have to look through the cumulative diff too.
2) Fair point, it was bothering me to see inconsistencies, but the best course of action may well be to ignore it. I'll close this PR until someone else is also bothered by the inconsistency.